### PR TITLE
Adds screenshots API

### PR DESCRIPTION
**Pull request by proxycrawl**
The free tier is 50 requests with all the functionality enabled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1199,6 +1199,7 @@ Table of Contents
 
 ## Screenshot APIs
 
+  * [ProxyCrawl.com](https://proxycrawl.com/screenshots-api) - Automate website screenshots while avoiding captchas and blocks with an API.
```
Comment (ripienaar) [inline README.md:1202]: please elaborate on what is the free plan for this in the description

**Review by ripienaar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (proxycrawl) [reply README.md]: @ripienaar done

File: README.md [hunk 2]
```diff
@@ -1199,6 +1199,7 @@ Table of Contents
 
 ## Screenshot APIs
 
+  * [ProxyCrawl.com](https://proxycrawl.com/screenshots-api) - Automate website screenshots while avoiding captchas and blocks with an API. Free tier includes 50 screenshots with all the available functionality.
```
Comment (ripienaar) [inline README.md:1202]: Is this 50 per month forever? I couldnt figure this out on the page

**Comment by proxycrawl** [comment]
@ripienaar done, we've added a clarification

File: README.md [hunk 3]
```diff
@@ -1199,6 +1199,7 @@ Table of Contents
 
 ## Screenshot APIs
 
+  * [ProxyCrawl.com](https://proxycrawl.com/screenshots-api) - Automate website screenshots while avoiding captchas and blocks with an API. Free tier includes 50 screenshots (one time, not monthly) with all the available functionality.
```
Comment (ripienaar) [inline README.md:1202]: so its a one off 50 screenshots free? That would unfortunately not qualify for this list I am afraid. It just isnt a free tier, compare to the requirement where free services offered has to be for at least a year this just doesnt cut it.

If it was like 5 or 10 a month, great, as is afraid its not enough for this list

**Pull request closed without merging**